index.bs

```
@@ -578,48 +578,39 @@ When the <dfn method for="XRFrame">getViewerPose(|referenceSpace|)</dfn> method
 
 </div>
 
-<div class="algorithm" data-algorithm="get-input-pose">
+<div class="algorithm" data-algorithm="get-pose">
 
-When the <dfn method for="XRFrame">getInputPose(|inputSource|, |referenceSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
+When the <dfn method for="XRFrame">getPose(|space|, |relativeTo|)</dfn> method is invoked, the user agent MUST run the following steps:
 
   1. If the {{XRFrame}}'s [=active=] boolean is <code>false</code>, throw a {{InvalidStateError}} and abort these steps.
   1. Let |session| be the {{XRFrame}}'s {{XRFrame/session}} object.
-  1. If |referenceSpace|'s [=XRSpace/session=] does not equal |session|, return <code>null</code> and abort these steps.
-  1. If |inputSource|'s pose cannot be determined relative to |referenceSpace|, return <code>null</code>
-  1. Return a new {{XRInputPose}} describing |inputSource|'s pose relative to the origin of |referenceSpace|.
+  1. If |space|'s [=XRSpace/session=] does not equal |session|, return <code>null</code> and abort these steps.
```

This seems like it should throw rather than return null.  wdyt?

---

Same for the next line.